README.md

```diff
@@ -27,18 +27,19 @@ Also use an app for creating data that should be presented in receipt.
 
 ##### Manual
 
+- Open `src/backend/Views/Receipt/receipt.cshtml` and change the `link` and `script` tags according to the comments in that file.
```

Comment:
Could you leave this as is if you want to run whatever js is available in master or do you need to switch to the localhost always ?

Reply:
I figured it out. Overlooked the css I needed to comments in, so this should always be updated